packages/components/src/components/radio-group/radio-group.jsx

```diff
@@ -35,7 +62,7 @@ RadioGroup.propTypes = {
         })
     ),
     onToggle: PropTypes.func,
-    selected: PropTypes.bool,
+    defaultOption: PropTypes.any,
```

Comment:
Could we rafctor this to `default_option` if it's a variable? Same for other variables.

Reply:
Mamali mentioned above and I solved.